README.md

```diff
@@ -80,9 +82,12 @@ The library leverages the following environment variables:
 
 - Store any Jinja2 variable definitions you wish to use in your configuration
   template files in `resources/myapp.yml`.
-- Store your `docker-compose.yml.j2` file under `resources/templates/cli/`.
```

Comment:
Now is probably a good time to move the location of this. I don't think we should insist of a `cli` folder. Let's place the `docker-compose.yml.j2` file just at `resources/templates/baseline/` (we'll need to update our code which copies it into image and refers to it).
Also update the path in the python block above please